APE10.rst

```diff
@@ -0,0 +1,178 @@
+Roadmap for Python 3-only support
+---------------------------------
+
+author: Tom Robitaille
+
+date-created: 2016 July 13
+
+date-last-revised: 2016 July 13
+
+type: Process
+
+status: Draft
+
+Abstract
+--------
+
+The purpose of this APE is to set out a roadmap for future releases of the
+Astropy core package, in particular to identify the process for switching to
+releases that support only Python 3.
```

Comment:
Maybe also add here also the specific fact of "ending support for py 2.x in December 2019"